src/utils/experimentUpdatesHandler.js

```diff
@@ -94,11 +94,21 @@ const uglyTemporalFixedProcessingConfig = (processingConfig, filterName) => {
   return fixedProcessingConfig;
 };
 
-const onGEM2SUpdate = (update, dispatch) => {
+const onGEM2SUpdate = (experimentId, update, dispatch) => {
   const processingConfig = update?.item?.processingConfig;
   if (processingConfig) {
     let fixedProcessingConfig = uglyTemporalFixedProcessingConfig(processingConfig, 'classifier');
     fixedProcessingConfig = uglyTemporalFixedProcessingConfig(fixedProcessingConfig, 'mitochondrialContent');
+
+    // adding default config to every filter with auto option
+    Object.keys(fixedProcessingConfig).forEach((key) => {
```

Comment:
Actually maybe i should do this in the api.

Reply:
I think it's better to have it in the UI, as this is temporary data.